gm4_combat_expanded/data/gm4_combat_expanded/functions/item_modify_eval/chest_update.mcfunction

```diff
@@ -0,0 +1,2 @@
+
+$item modify entity @s armor.chest {function:"minecraft:set_components",components:$(components)}
```

Comment:
All of these `item_modify_eval` functions are missing a header comment with info about where it's called from.